packages/handler-kit/src/http/request.ts

```diff
@@ -29,3 +38,46 @@ export const request = (url: string): HttpRequest => ({
   headers: {},
   body: undefined,
 });
+
+// TODO: write a better multipart/form-data parser
+// that parse also the meta informartion (name and other key value fields)
+export const parseMultipart = (req: HttpRequest) =>
+  pipe(
+    sequenceS(E.Apply)({
+      boundary: pipe(
+        req.headers,
+        lookup("content-type"),
+        E.fromOption(
+          () => new HttpBadRequestError("missing content-type header")
+        ),
+        E.filterOrElse(
+          (contentType) => contentType.includes("multipart/form-data"),
+          () =>
+            new HttpBadRequestError(
+              "the content-type is not multipart/form-data"
+            )
+        ),
+        E.chain(
+          flow(
+            multipart.getBoundary,
```

Comment:
Perhaps better to add a tryCatch since an exception may be thrown

Reply:
i already checked: the library code from `parse-multipart` doesn't throws, instead it returns empty results: empty string in the case of the boundary (that is checked in the `E.filterOrElse`) and empty array on `Parse` that it's fine and should be checked at application level (on body decode phase)

Reply:
I added a `try catch` guard to the external lib